js/components/graph/Edge.js

```diff
@@ -5,14 +5,14 @@ import React from "react"
  * Function-based component representing an edge from a Node/Bool to a Node/Bool
  */
 export default function Edge(props) {
-  const pathAttrs = { d: "M" }
+  let pathDescription = "M"
   props.points.forEach(p => {
-    pathAttrs.d += p[0] + "," + p[1] + " "
+    pathDescription += p[0] + "," + p[1] + " " + " "
```

Comment:
There seems to be an additional `" "` which is unnecessary, and causing some of the snapshot tests to fail. (Remember you should run the Javascript tests locally with `yarn run test`)